packages/frontend/core/src/desktop/dialogs/setting/general-setting/editor/general.tsx

```diff
@@ -351,7 +355,7 @@ const NewDocDefaultModeSettings = () => {
   );
 };
 
-export const DeFaultCodeBlockSettings = () => {
+export const DefaultCodeBlockSettings = () => {
```

Comment:
?

Reply:
I thinks this can be removed.
should be put in 'Editor' settings if new requirement comes